cmake/thirdparty/get_legion.cmake

```diff
@@ -131,6 +134,8 @@ function(find_or_configure_legion)
       list(APPEND CMAKE_C_IMPLICIT_LINK_DIRECTORIES "${_cuda_stubs}")
       list(APPEND CMAKE_CXX_IMPLICIT_LINK_DIRECTORIES "${_cuda_stubs}")
       list(APPEND CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES "${_cuda_stubs}")
+      set(legate_core_cuda_stubs_path "${_cuda_stubs}" PARENT_SCOPE)
+      set(legate_core_cuda_stubs_path "${_cuda_stubs}" CACHE STRING "" FORCE)
```

Comment:
Sorry - this is probably obvious - but why do we set a regular variable in the parent scope and a cache variable?

Reply:
I'm being defensive in case a reconfigure of the build dir causes this code path not to be. The package config file will still be written, and I don't want to drop the implicit include lines.